src/include/kdbopmphm.h

```diff
@@ -22,17 +22,25 @@
  * opmphmRatio should never be less than 1.
```

Comment:
Btw. of course is getting the code correct, fast, and elegant much more important than these little things. This is why I merged it immediately. Simply continue your work on the main parts...